docs/download.rst

```diff
@@ -0,0 +1,28 @@
+.. _bundle-label:
+
+Download
+========
+
+Download the latest HyperSpy-bundler installer:
+
+.. raw:: html
+
+    <div class="text-center">
+        <a  class="downloadbutton"
+            href="https://github.com/hyperspy/hyperspy-bundle/releases/latest">
+                Download HyperSpy-bundle installer
+        </a>
+    </div>
+    <br>
+
+
+Depending on your computer, choose the most suitable installer:
+
+.. figure:: _static/bundle_release_download.png
```

Comment:
I'll send a PR to correct `RELEASE_TEXT.md`. This figure would need to be updated accordingly.

Reply:
See #68